src/azul/indexer.py

```diff
@@ -119,9 +126,11 @@ def _get_bundle(self, bundle_uuid, bundle_version):
         assert _ == bundle_version
         return manifest, metadata_files
 
-    def _add_test_modifications(self, manifest, metadata_files, dss_notification):
+    def _add_test_modifications(self, manifest, metadata_files, dss_notification, default):
```

Comment:
`default` is not very descriptive. But see comment at call site.
```suggestion
    def _add_test_modifications(self, bundle_uuid, manifest, metadata_files, dss_notification):
```

Reply:
done